changelogs/unreleased/2456-increase-log-level-modules-update-command.yml

```
@@ -0,0 +1,7 @@
+description: Increase log level to INFO when compiler service is updating modules
```

This PR changes the log level to `DEBUG` instead of `INFO`. I think that's a good idea, because that way the behavior on iso4 is consistent with the behavior on iso5+. So updating the changelog message is sufficient in my opinion.
```suggestion
description: Increase log level to DEBUG when compiler service is updating modules
```